spec/requests/transformation_mappings_spec.rb

```diff
@@ -305,6 +305,79 @@ def href_slug(obj)
           expect(updated_mapping_destination).to match_array(reference_destination)
           expect(transformation_mapping.transformation_mapping_items.count).to eq(TransformationMappingItem.all.count)
         end
+
+        context "can validate vms with csv data and service_template_id are specified" do
+          it "vm belongs to the service_template record" do
+            api_basic_authorize(action_identifier(:transformation_mappings, :validate_vms, :resource_actions, :post))
+            source_ems = FactoryGirl.create(:ems_cluster)
+            destination_ems = FactoryGirl.create(:ems_cluster)
+            transformation_mapping =
+              FactoryGirl.create(:transformation_mapping,
+                                 :transformation_mapping_items => [TransformationMappingItem.new(:source => source_ems, :destination => destination_ems)])
+            vm = FactoryGirl.create(:vm_vmware, :name => 'test_vm', :ems_cluster => source_ems, :ext_management_system => FactoryGirl.create(:ext_management_system))
```

Comment:
Please don't set `name` as there's a sequencer for it.

Reply:
Should one of the clusters above (assuming it's supposed to be a cluster) be connected to this `ext_management_system`?